Gemfile

```diff
@@ -13,10 +13,11 @@ end
 group :development do
   gem 'activesupport', '~> 4.0'
   gem 'builder'
+  gem 'bump',          '~> 0.6'
   gem 'bundler',       '~> 1.12'
   gem 'guard-rspec'
   gem 'hobo_support'
-  gem 'invoca-utils',  '~> 0.0.3'
+  gem 'invoca-utils',  '~> 0.1'
```

Comment:
Below, can you remove the version specs from `rake` and `rspec`, the run `bundle update rake rspec`. That will get us the latest, which is what we want. Especially for `rake` since it actually has a CVE out.